Apps/MaybeFinance/docker-compose.yml

```
@@ -0,0 +1,274 @@
+
+    # Container restart policy
+    restart: unless-stopped
+
+    # Privilege of the container
+    privileged: false
+
+    # Uses default network mode
+    networks:
+      # Use the default Docker network
+      - default
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)